src/index.js

```diff
@@ -1,43 +1,32 @@
 const fs = require('fs-extra')
 const { join } = require('path')
-const { promisify } = require('util')
 const _ = require('lodash')
 const {
     AGENT_LANGS,
     generateWrapperCode,
     generateWrapperExt,
 } = require('./handlers')
 
-const mkdir = promisify(fs.mkdir)
-const writeFile = promisify(fs.writeFile)
 
 const VALIDATE_LIB_BY_LANG = {
     /**
      * Validates the python Thundra's library
      */
     python() {
-        this.log(
-            'Please ensure that all necessary Thundra Python agents are installed'
-        )
+        this.log('Please ensure that all necessary Thundra Python agents are installed')
     },
     /**
      * Validates the node Thundra's library
      */
-    async node() {
-        let pack
+    node() {
         try {
-            pack = await fs.readJson(join(this.prefix, 'package.json'))
-        } catch (err) {
-            this.log(
-                'Could not read package.json. Skipping Thundra library validation - please make sure you have it installed!'
-            )
-            return
-        }
-        const { dependencies = [] } = pack
-        if (!Object.keys(dependencies).some(dep => dep === '@thundra/core')) {
-            throw new Error(
-                "Thundra's Node library must be installed in order to use this plugin!"
-            )
+            pack = fs.readJsonSync(join(this.prefix, 'package.json'))
+            const { dependencies = [] } = pack
+            if (!Object.keys(dependencies).some(dep => dep === '@thundra/core')) {
+                throw new Error("Thundra's Node library must be installed in order to use this plugin!")
+            }
+        } catch(err) {
+            this.log('Could not read package.json. Skipping Thundra library validation - please make sure you have it installed!')
```

Comment:
This also catches the error from line `26` and it shows different error message instead of the thrown error's message above. Also we are swallowing that error here

Reply:
Right, fixed it 👍